charts/osm/values.yaml

```diff
@@ -44,6 +45,9 @@ OpenServiceMesh:
   useHTTPSIngress: false
   envoyLogLevel: error
   enforceSingleMesh: false
+  customLogAnalyticsWorkspace: true
+  fluentbitOutput: stdout
+  webhookName: osm-webhook
```

Comment:
this changes the webhookName, previously it was of the form `osm-webhook-<meshName>`